packages/js/src/dashboard/components/info-tooltip.js

```diff
@@ -9,11 +9,11 @@ import { InformationCircleIcon } from "@heroicons/react/outline";
  * @returns {JSX.Element} The element.
  */
 export const InfoTooltip = ( { children } ) => (
-	<TooltipContainer className="yst-h-fit yst-leading-[0]">
+	<TooltipContainer as="div" className="yst-h-fit yst-leading-[0]">
 		<TooltipTrigger>
 			<InformationCircleIcon className="yst-w-5 yst-h-5 yst-text-slate-400" />
 		</TooltipTrigger>
-		<TooltipWithContext variant="light" className="yst-max-w-80 yst-p-4" position="left">
+		<TooltipWithContext variant="light" className="yst-max-w-80 yst-p-4 yst-shadow-md" position="left">
```

Comment:
See my comment on `yst-leading-4` in the `WidgetDataSources`. I think it should be `normal` though?

```suggestion
		<TooltipWithContext variant="light" className="yst-leading-normal yst-max-w-80 yst-p-4 yst-shadow-md" position="left">
```